src/base/bittorrent/peerinfo.cpp

```diff
@@ -51,8 +51,8 @@ PeerAddress::PeerAddress(const QHostAddress &ip, ushort port)
 
 // PeerInfo
 
-PeerInfo::PeerInfo(const TorrentHandle *torrent, const libt::peer_info &nativeInfo)
-    : m_nativeInfo(nativeInfo)
+PeerInfo::PeerInfo(const TorrentHandle *torrent, libt::peer_info nativeInfo)
+    : m_nativeInfo(std::move(nativeInfo))
```

Comment:
Sorry, I don't understand this change. And all the similar. What's it for?

Reply:
I read somewhere that it costs 2 moves instead of 1 copy and 1 move but turns out it's for rvalues... i'll just drop this commit.

Reply:
>i'll just drop this commit.

👍